docs/coordinates/remote_methods.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-remote:
+
+Methods that access remote resources
+------------------------------------
+
+There are currently two methods that rely on getting remote data to work.
+
+The first is the :class:`~astropy.coordinates.SkyCoord` :meth:`~astropy.coordinates.SkyCoord.from_name` method, which uses
+`Sesame <http://cds.u-strasbg.fr/cgi-bin/Sesame>`_ to retrieve coordinates
+for a particular named object::
+
+    >>> SkyCoord.from_name("M42")  # doctest: +REMOTE_DATA +FLOAT_CMP
+    <SkyCoord (ICRS): (ra, dec) in deg
+        (83.82208, -5.39111)>
+
+The second is the :class:`~astropy.coordinates.EarthLocation` :meth:`~astropy.coordinates.EarthLocation.of_site` method, which
+provides a similar quick way to get an
+:class:`~astropy.coordinates.EarthLocation` from an observatory name::
+
+    >>> from astropy.coordinates import EarthLocation
+    >>> EarthLocation.of_site('Apache Point Observatory')  # doctest: +REMOTE_DATA +FLOAT_CMP
+    <EarthLocation (-1463969.3018517173, -5166673.342234327, 3434985.7120456537) m>
+    
+The full list of available observatory names can be obtained with
+the :class:`~astropy.coordinates.EarthLocation` :meth:`~astropy.coordinates.EarthLocation.get_site_names` method.
+
+While these methods are convenient, there are several considerations to take into account:
+
+* Since these methods access online data, the data may evolve over time (for
+  example, the accuracy of coordinates might improve, and new observatories
+  may be added). However, this means that a script using these and running
+  now may give a different answer in five years. Therefore, users concerned
+  with reproducibility should not use these methods in their final scripts,
+  but can instead use them to get the values required and then hard-code them
+  into the scripts. For example, we can check the coordinates of the Kitt
+  Peak Observatories using::
+
+    >>> loc = EarthLocation.of_site('Kitt Peak')
```

Comment:
Maybe add a comment at the end of this line that says "note that this uses the internet".  You also may need the REMOTE_DATA doctest marker